maker/src/actor_system.rs

```diff
@@ -177,30 +231,55 @@ where
         });
 
         let (listener_supervisor, listener_actor) = Supervisor::<_, listener::Error>::with_policy(
-            move || listener::Actor::new(endpoint_addr.clone(), listen_multiaddr.clone()),
+            {
+                let listen_multiaddr = listen_multiaddr.clone();
+                let endpoint_addr = endpoint_addr.clone();
+                move || listener::Actor::new(endpoint_addr.clone(), listen_multiaddr.clone())
+            },
             always_restart_after(RESTART_INTERVAL),
         );
 
+        // TODO: Shouldn't this actor also be supervised?
         let pong_address = pong::Actor.create(None).spawn(&mut tasks);
 
+        let listen_protocols = ListenProtocols::default();
+        let (identify_listener_supervisor, identify_listener_actor) = Supervisor::new({
+            let identity = identity.libp2p.clone();
+            move || {
+                identify::listener::Actor::new(
+                    daemon::version::version().to_string(),
+                    Environment::Maker,
```

Comment:
🤔 Just say `Environment::Unknown` if we don't want to bother with makers sharing this information? Adding a dedicated variant for makers seems very strange.

Reply:
Fair enough, I don't really care much for now 😅
If we ever want to use this for the maker it will require some more design.